src/thermopro_ble/parser.py

```diff
@@ -41,9 +44,15 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             return
         if len(list(service_info.manufacturer_data.values())[0]) < 4:
             return
-        self.set_device_type(name.split(" ")[0])
-        self.set_title(f"{name} {short_address(service_info.address)}")
-        self.set_device_name(name)
+        model = name.split(" ")[0]
+        self.set_device_type(model)
+        if model.startswith("TP96"):
+            # This series can have multiple probes, so we must assume probe 1
+            self.set_title(f"{name} Probe 1 {short_address(service_info.address)}")
+            self.set_device_name(f"{name} PROBE 1")
+        else:
+            self.set_title(f"{name} {short_address(service_info.address)}")
+            self.set_device_name(name)
```

Comment:
The title / device name is for the whole device so we want to keep it as-is. I'll merge this PR and do a cleanup on the original PR